src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -369,15 +339,7 @@ def __call__(
 
                               Default value: False
         :type share_outputs: bool, optional
-        :param shared_memory: Deprecated. Works like `share_inputs` mode.
-
-                              If not specified, function uses `share_inputs` value.
-
-                              Note: Will be removed in 2024.0 release!
-                              Note: This is keyword-only argument.
 
-                              Default value: None
-        :type shared_memory: bool, optional
```

Comment:
It's not removed from parameters of function. Same for all other infer-functions.
https://github.com/openvinotoolkit/openvino/pull/21675/files#diff-10e55df1b8f3b6e1926cee550ab907b024b095f2b5f4d26efd3c7a97f08ecb58R276

Reply:
removed